CF01/empty-fields-error.js

```diff
@@ -0,0 +1,49 @@
+const { beforeEach } = require("mocha");
+
+describe ('Empty Fields', function(){
+    beforeEach(function() {
+        cy.visit('https://qa-angular-conduit.vercel.app/register'); 
+    })
+    afterEach(function() {
+        cy.wait(1000);
+    })
+       
+    it('Empty Password',function(){
+        cy.get(':nth-child(1) > .form-control').type('santiago.lopez');
+        cy.get(':nth-child(2) > .form-control').type('santiago.lopez@wolax.com.ar');
+        cy.get('.btn').then((btn) => {
+            if (cy.get('.btn').should('have.attr','disabled')==false) {
```

Comment:
did you test this? I think the .should method doesn't work as you expect. 
try to test for both cases to make sure it works please.